README.md

```diff
@@ -671,6 +671,9 @@ post.apiDoc = {
 
 ```
 
+Alternatively, args.paths may contain route specifications of the form
```

Comment:
I realize we'll end up internally merging these with `fs-routes`; however, on second look we should probably stay more inline with openapi concepts.  How about something like this? `{path: '/foo/{id}', module: require('./handlers/foo')}`